db/migrate/20250421074012_create_category_localization_table.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateCategoryLocalizationTable < ActiveRecord::Migration[7.2]
+  def change
+    create_table :category_localizations do |t|
+      t.references :category, null: false, foreign_key: true
```

Comment:
```suggestion
      t.references :category, null: false
```

We currently do not encourage the use of foreign keys in core as we have yet to reach a conclusion about its trade-offs.